sites/hashdev/src/_pages/blog/1_block-design.mdx

```diff
@@ -5,7 +5,7 @@ title: "Design guidelines for building blocks with the Block Protocol"
 subtitle: "How to make your blocks feel intuitive, easy to use, and beautiful across every application."
 author: "Maggie Appleton"
 jobTitle: "Head of Design at HASH"
-date: "April 24th, 2022"
+date: "2022-04-24"
```

Comment:
@nathggns the reason I made this change is because I think this reduces the chance of inconsistent formatting between the blog post pages. It also lets us make use of `toLocaleString` to format the date, which could be handy for internationalisation.

Let me know if you agree with this thinking.